classes/system/kibana/server/single.yml

```diff
@@ -1,11 +1,12 @@
 classes:
 - service.kibana.server.single
 parameters:
-  _param:
-    kibana_elasticsearch_host: localhost
   kibana:
     server:
       enabled: true
+      bind:
+        address: ${_param:kibana_host}
```

Comment:
Can't you directly use `${_param:single_adress}` here? I think that's what we do for other services (InfluxDB, etc.)